tests/api/test_api.py

```diff
@@ -168,15 +168,16 @@ def test_is_node_schema_valid_skipped(cript_api: cript.API) -> None:
 
     """
 
-    cript_api.skip_validation = True
-    # ------ invalid node schema------
-    invalid_schema = {"invalid key": "invalid value", "node": ["Material"]}
+    with cript.API(host=cript_api.host, api_token=cript_api._api_token, storage_token=cript_api._storage_token) as local_cript_api:
+        local_cript_api.skip_validation = True
```

Comment:
I don't love this because we are repeating code and it looks a bit weird, but we can add this as a tech debt into our trello and refactor and upgrade this after this release, if that works with you

Reply:
I see.
It is necessary to use a local API instance here. Otherwise we are changing the API for every other test as well.
There is a way to use `function scope` on the pytest fixture that would work too.